src/cursor.rs

```diff
@@ -108,6 +109,7 @@ macro_rules! implement(
                     Ok(value) => {
                         value.map(|values| Ok(Row {
                             column_mapping: self.statement.column_mapping(),
+                            column_names: self.statement.column_names_rc(),
```

Comment:
Please move it before `column_mapping` and just `self.statement.column_names.clone()`.

Reply:
Done